### ISSUE TITLE
exp_average_loss 

### ISSUE THREAD

**Issue by leejason** [cybertronai/bflm#3]
Is it possible to why using the following? 

`exp_average_loss = loss.item() if exp_average_loss is None else 0.7*exp_average_loss+0.3*loss.item()
`

**Comment by 8enmann** [comment]
This is just to make logging more stable.
https://en.wikipedia.org/wiki/Moving_average#Exponential_moving_average
